src/core/public/notifications/toasts/global_toast_list.tsx

```diff
@@ -53,9 +53,10 @@ export class GlobalToastList extends React.Component<Props, State> {
   public render() {
     return (
       <EuiGlobalToastList
+        data-test-subj="globalToastList"
         toasts={this.state.toasts}
         dismissToast={this.props.dismissToast}
-        toastLifeTimeMs={6000}
+        toastLifeTimeMs={Infinity}
```

Comment:
What's the reason behind changing the default? 6000 (or something < Infinity) feels like a safer default to prevent a toasts from hanging around if this wasn't the intention.

Reply:
@timroes Do you have thoughts here?

Reply:
Talked to Tim about this, the value here doesn't matter too much since all toasts get the info timeout by default. Leaving as `Infinity` will make functional tests fail to alert the developer that they have not specified a timeout (if a new type is added in the future that doesn't get the info timeout somehow). I'll add a comment.